docs/member/fixed-income.md

```diff
@@ -101,7 +101,7 @@ Click on know more to get detailed information of the bond.
 
 ## 4. How To Buy a security?
 
-It's crucial to update your KYC, bank information, and demat account information in the Manage Account section in the sidebar navigation before you Buy a security.
+It is crucial to update your KYC, bank information, and demat account information in the Manage Account section in the sidebar navigation before you buy a security. in the sidebar navigation before you sell a security.
```

Comment:
`It is crucial to update your KYC, bank information, and demat account information in the Manage Account section in the sidebar navigation before you buy a security. in the sidebar navigation before you sell a security.`

please revisit this and change as per what you think is correct